webapp/src/main/webapp/src/app/shared/base.service.ts

```diff
@@ -25,4 +25,8 @@ export abstract class BaseService<T extends BaseEntity> {
 		return this.httpClient.delete<T>(environment.apiBaseUrl + resource + '/' + entity.id);
 	}
 
+	searchCountAndItems(resource: string, paramsData: any): Observable<BaseCountAndItems<T>> {
```

Comment:
I said that you should not use `any` here, use a concrete type

Reply:
That is your comment Wyclif "Then in that case limit the types to string and undefined, typescript supports this i.e. multiple types for a single argument"

Reply:
That was for the other function, not this function, a saying for the second argument please use a concrete type and not any, use the same type as the ng resource function you are calling from here

Reply:
Which one?